Cubical/Tactics/Reflection.agda

```diff
@@ -89,7 +89,7 @@ get-boundary tm = unapply-path tm >>= λ where
 equation-solver : List Name → (Term -> Term -> TC Term) → Bool → Term → TC Unit
 equation-solver don't-Reduce mk-call debug hole =
     withNormalisation false (
-    dontReduceDefs don't-Reduce (
```

Comment:
This should at least be a warning with 2.6.4

Reply:
But this file isn't in 0.5 :)

Reply:
Ah, right!